### PR TITLE
remove some unused variables

### PR DESCRIPTION
**Pull request by fchapoton**
found using 
`ruff check --select=F841 --output-format=concise src/sage/`

### :memo: Checklist

- [x] The title is concise and informative.
- [x] The description explains in detail what this PR is about.
- [ ] I have linked a relevant issue or discussion.
- [ ] I have created tests covering the changes.
- [ ] I have updated the documentation and checked the documentation preview.

### CODE REVIEW

File: src/sage/geometry/lattice_polytope.py [hunk 1]
```diff
@@ -591,7 +591,7 @@ def _sage_input_(self, sib, coerced):
         """
         if self._ambient is not self:
             raise NotImplementedError
-        data = self._vertices
+        _ = self._vertices
```
Comment (dcoudert) [inline src/sage/geometry/lattice_polytope.py]: is this call needed ?

**Review by dcoudert — COMMENTED** [review]
otherwise LGTM.

File: src/sage/geometry/lattice_polytope.py (hunk 1 above)
Comment (fchapoton) [reply src/sage/geometry/lattice_polytope.py]: good point. I am not sure.

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by fchapoton** [comment]
so positive review ?

**Comment by dcoudert** [comment]
yes

**Pull request merged**